buildspecs/linux_arm.spec

```diff
@@ -136,6 +136,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="build_dropToHursley" value="true"/>
 		<flag id="build_j2se" value="true"/>
 		<flag id="build_java8" value="true"/>
+		<flag id="build_jitserver" value="true"/>
```

Comment:
I don't expect build_jitserver should be set true for this platform.

Reply:
Right, fixed it.